File: DSA/VaniAgrawal_590015846/Day_4_Question_2.py
#Rotate the array to the right by k steps and return the rotated array.
nums=list(map(int,input("enter list with space :").split()))
k= int(input("enter by  how many values you want to rotate :"))
i=1
while i<=k:
    last_no=nums[-1]
    nums.insert(0,last_no)
    nums.pop()
    i=i+1

    
print(nums)

File: DSA/VikrantSinghBanwala_590018082/day4_question2.c
#include<stdio.h>
void rotate(int* nums,int n,int k){
    k=k%n;
    int temp[n];
    for(int i=0;i<n;i++){
        temp[(i+k)%n]=nums[i];
    }   
    for(int i=0;i<n;i++){
        nums[i]=temp[i];
    }
}
int main(){
    int n,k;
    printf("Enter the size of the array: ");
    scanf("%d",&n);    
    int nums[n];
    printf("Enter the elements of the array: ");
    for(int i=0;i<n;i++){
        scanf("%d",&nums[i]);
    }
    printf("Enter the number of steps to rotate: ");
    scanf("%d",&k);    
    rotate(nums,n,k);
    printf("The rotated array is:");
    for(int i=0;i<n;i++){
        printf("%d",nums[i]);
    }
    printf("\n");
    return 0;
}